docs/artefacts/codecs.md

```diff
@@ -97,3 +97,37 @@ For **gauge** there is also the field `action` which might be `add` if the value
 ### yaml
 
 En- and decodes [YAML](https://yaml.org).
+
+### syslog
+
+En- and decodes syslog messages (both, the standard IETF format and the old BSD format).
+A typical syslog message as follows:
```

Comment:
Please show the formats for both standards, and if possible, link the RFCS.

It is also worth a mention that an invalid syslog message, will still be parsed, but the whole string will be put into the `msg` field.